Mathlib/Topology/Compactification/OnePointEquiv.lean

```diff
@@ -0,0 +1,70 @@
+/-
+Copyright (c) 2024 Bjørn Kjos-Hanssen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bjørn Kjos-Hanssen, Oliver Nash
+-/
+import Mathlib.LinearAlgebra.Projectivization.Basic
+import Mathlib.Topology.Compactification.OnePoint
+
+/-!
+# One-point compactification and projectivization
+
+We construct a set-theoretic equivalence between
+`OnePoint K` and the projectivization `ℙ K (K × K)` for an arbitrary division ring `K`.
+
+TODO: Add the extension of this equivalence to a homeomorphism in the case `K = ℝ`,
+where `OnePoint ℝ` gets the topology of one-point compactification.
+
+
+## Main definitions and results
+
+* `equivProjectivization` : the equivalence `OnePoint K ≃ ℙ K (K × K)`.
```

Comment:
Fully qualifying names in doc comments makes links work in our [HTML docs](https://leanprover-community.github.io/mathlib4_docs/)
```suggestion
* `OnePoint.equivProjectivization` : the equivalence `OnePoint K ≃ ℙ K (K × K)`.
```